synapse/cores/common.py

```diff
@@ -1938,3 +1912,68 @@ def _tufosByInetCidr(self, prop, valu, limit=None):
 
         return self.getTufosBy('range', prop, (ipv4addr, ipv4addr+mask), limit=limit)
 
+    #############################################################
```

Comment:
Cortex is-a datamodel is only a couple of months old; see https://github.com/vivisect/synapse/commit/327e3c31bbd3f0950335b601c672fa69b5f0e490, it was previously has-a.